code-challenges.js

```diff
@@ -9,38 +9,142 @@
+//         myNumber.push(number[i] * 5)
+//     }
+//     return myNumber
+// }
+// console.log(multipliedByFive(myNumbers1))
+//Done
 
 
 
 // --------------------3) Create a function that takes in an array of numbers and returns an array containing only the odd numbers.
 // Use the test variable provided below. Expected output: [-7, 9, 13]
 
-var myNumbers2 = [8, -7, 0, 6, 2, 9, 13]
-
-
+// var myNumbers2 = [8, -7, 0, 6, 2, 9, 13]
+//Declare a function
+//Input: takes in a list or an array of numbers
+//Output: returns only the odd list of the array  => [-7, 9, 13]
+//I need to loop throught the array and check if the number is odd or even
+//If odd then print in my new array list
+//If divisbale my 0, then do not print
+//call the function
+
+// const oddNumFunction = (num) =>{
+//     let arrayNum = []
+//     for(let i = 0; i<num.length; i++){
+//         if(num[i] % 2 !== 0){
+//             arrayNum.push(num[i])
+//         }
+//     }
+//     return arrayNum
+// }
+// console.log(oddNumFunction(myNumbers2))
+//Done
 
 
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
-
+// var stringWithVowels1 = "HeyThereLearnStudent"
+// var stringWithVowels2 = "ILoveJavaScript"
+
+// //Create a function
+// //That removes all the vowels from a string
+// //make the strings into an array using stringWithVowels1.split
+// //then put it back together into a sting 
+
+// const takeOutVowels = (strings) =>{
+
+//     let myNewString = ""
+//     for(let i = 0; i<strings.length; i++){
+//     //  console.log(strings[i])
+//     //If aeiou take it out
+//     let lowerCase = strings[i].toLowerCase()
+//     if(lowerCase !== "a" && lowerCase  !== "e"  && lowerCase  !== "i"  && strings[i] !== "o"  && strings[i] !== "u"){
+//         myNewString+=strings[i]
+//     }
+//     // else if(strings[i] !== "e"){
+//     //     myNewString+=strings[i]
+//     // }
+//     // else if(strings[i] !== "i"){
+//     //     myNewString+=strings[i]
+//     // }
+//     // else if(strings[i] !== "o"){
+//     //     myNewString+=strings[i]
+//     // }
+//     // else if(strings[i] !== "u"){
+//     //     myNewString+=strings[i]
+//     // }
+//   }
+// return  myNewString
+// }
+// console.log(takeOutVowels(stringWithVowels1))
+// console.log(takeOutVowels(stringWithVowels2 ))
+
+
+
+// const takeOutVowels = (strings) =>{
+//     let myNewString = ""
+//     for(let i = 0; i<strings.length; i++){
+//     //  console.log(strings[i])
+//     //If aeiou take it out
+//     if(strings[i] !== "a" && strings[i] !== "e"  && strings[i] !== "i"  && strings[i] !== "o"  && strings[i] !== "u"){
+//         myNewString+=strings[i]
+//     }
+//   }
+// return  myNewString
+// }
+// console.log(takeOutVowels(stringWithVowels2))
```

Comment:
I'm glad you tried this two different ways. I think the second solution is pretty close. I think if you combined lines 112 and 113 into the second approach it would be perfect.